### PR TITLE
adding english to language attribute of html tag

### PR DESCRIPTION
**Pull request by kstack7**
#127 for reference

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang='en-US'>
```
Comment (juliaelman) [inline app/views/layouts/application.html.erb]: Would you mind changing this to use double quotations for consistency?

**Comment by kstack7** [comment]
Sure! One sec...

**Comment by kstack7** [comment]
@juliaelman all set

**Comment by juliaelman** [comment]
Awesome! Thank you @kstack7! You :crown:

**Pull request merged**